funnel/templates/space.html

```diff
@@ -42,6 +42,13 @@
                 {%- if proposal.user != proposal.speaker %}
                   <small>via {{ proposal.user.pickername }}</small>
                 {%- endif %}
+                {%- if g.user and 'edit-space' in g.permissions %}
+                  {%- if proposal.location_geonameid != space.location_geonameid %}
+                    <i class="fa fa-plane" title="Outstation Speaker - {{ proposal.location }}"></i>
+                  {%- else %}
+                    <i class="fa fa-map-marker" title="Same location - {{ proposal.location }}"></i>
```

Comment:
Skip the title tags -- hints on hover are bad UI and don't work on touch screens. Add an aria label instead for screen readers.